apps/acrs/behaviours/limit-parents.js

```diff
@@ -0,0 +1,17 @@
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    locals.parentCount = '1';
+    if(req.sessionModel.get('referred-parents') !== undefined) {
```

Comment:
Here we're setting `parentCount` to one even when the `referred-parents` array is empty. Could this potentially mask a bug where the aggregation is empty but the `parent-details` page will display it as 1?

Reply:
I reasoned that the first parent to be added is parent 1, and this information is needed before the aggregate receives parent-1's information. It is effectively a default here which is replaced if the aggregate array has any defined content or length.

I suppose this could be replaced with a switch where the default returned is '1' - or did you have any other thoughts?

Reply:
I guess it depends if `parentCount` must be displayed before aggregate is initialised. When parentCount doesnt reflect the true number of added parents is just a little confusing should there be a problem later.